### ISSUE TITLE
Is the code of LUDT also involved in this repo?

### ISSUE THREAD

**Issue by Jay-Ye** [594422814/UDT#4]
Bravo work! Is the code of LUDT also involved in this repo?  Because there seems only the conference version of UDT are in the folder tracker.

**Comment by 594422814** [comment]
Based on UDT, LUDT processes training data using image entropy and the KCF tracker. Besides, LUDT also involves the multi-trajectory loss for training. Please refer to the "/train/training" folder.

**Comment by 594422814** [comment]
Based on UDT, LUDT processes training data using image entropy and the KCF tracker. Besides, LUDT also involves the multi-trajectory loss for training. Please refer to the "/train/training" folder.

**Comment by Jay-Ye** [comment]
> Based on UDT, LUDT processes training data using image entropy and the KCF tracker. Besides, LUDT also involves the multi-trajectory loss for training. Please refer to the "/train/training" folder.

Thx for your reply! By the way, is the pre-trained model of LUDT available? And would you please kindly explain how to run LUDT on a test sequence?

**Comment by 594422814** [comment]
Please refer to 'tracker/UDT/demo/demo_UDT.m' for more details. The pretrained models are available at 'tracker/UDT/runfiles'.

**Comment by Jay-Ye** [comment]
> Please refer to 'tracker/UDT/demo/demo_UDT.m' for more details. The pretrained models are available at 'tracker/UDT/runfiles'.

Many thanks for the early reply, that's really a fantastic job! So the main difference between UDT (+) and LUDT (+) is the training process, UDT and LUDT share the same inference step, is that right?

**Comment by 594422814** [comment]
Yes. LUDT better preprocesses the training data and involves more training losses, and thus slightly improves the results. Their online inference stages are the same.

**Comment by Jay-Ye** [comment]
Thx for your help and patience, can't wait to see more novel works from you guys!
